README.md

```diff
@@ -290,6 +294,14 @@ global:
       some_int: 1234
       some_float: 3.14159
       some_bool: false
+    key-mapping:
+      # key-mapping is used to replace the attribute key with different keys
+      - key: servertracer.http.responsecode
+        replacement: http.status_code
+      - key:  servertracer.http.responsephrase
+        replacement: http.message
+        overwrite: true # replace attribute key even if the replacement string is already a key on the span attributes
```

Comment:
I am not sure I follow overwrite and keep -- why have two options? Either you override or you do not, right?

Reply:
Perhaps needs better naming/explanation, the idea is that since attributes are entries in a map/hashtable `keep` offers the option of preserving the original key-value pair independent of the choice to `overwrite` the original or not. I think you are seeing it as a single step operation and I am seeing as the two steps that happen with the operation in a map: add/update entry for new key -value pair according to `overwrite`, then decide what to do about the old key-value pair according to `keep`.

Makes sense?

Reply:
Clarified via offline conversation.